nubilanding/src/components/blog/form.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import styles from './styles.module.scss';
+
+class Form extends React.Component {
+  state = { email: '' };
+  constructor(args) {
+    super(args)
+    this.state = {
+
+    }
+  }
+
+  onChange(e) {
+    this.setState({
+      [e.target.email]: e.target.value
+    })
+  }
+  render() {
+    return (
+      <form className={styles.formContenedor}>
+        <input
+          type="email"
+          name="email"
+          placeholder="Dejanos tu email"
+          value={this.state.email}
+          onChange={this.onChange.bind(this)}
+          className={styles.input}></input>
+        <botton type="submit" name="boton" className={styles.enviar}></botton>
```

Comment:
1. Puedes usar la notación `<elemento />` para elementos que no tienen contenido.
2. No es necesario o que hagas `this.onChange.bind(this)` ya que por defecto en componentes de clase, React pasa en los eventos la información de this.
3. El nombre del atributo `name` en el botón podría ser más descriptivo para que refleje más la acción del botón y su rol dentro del formulario. `boton-enviar` podría ser mejor.
```suggestion
        <input
          type="email"
          name="email"
          placeholder="Dejanos tu email"
          value={this.state.email}
          onChange={this.onChange}
          className={styles.input}  />
        <button type="submit" name="boton" className={styles.enviar} />
```